core/src/test/scala/unit/kafka/server/KafkaApisTest.scala

```diff
@@ -4385,7 +4386,12 @@ class KafkaApisTest extends Logging {
 
     val shareFetchRequest = new ShareFetchRequest.Builder(shareFetchRequestData).build(ApiKeys.SHARE_FETCH.latestVersion)
     val request = buildRequest(shareFetchRequest)
-    kafkaApis = createKafkaApis(overrideProperties = Map(KafkaConfig.NewGroupCoordinatorEnableProp -> "true", KafkaConfig.ShareGroupEnableProp -> "true"))
+    kafkaApis = createKafkaApis(overrideProperties = Map(
+      KafkaConfig.NewGroupCoordinatorEnableProp -> "true",
```

Comment:
In theory, it should not be necessary to set NewGroupCoordinatorEnableProp or ShareGroupEnableProp. The `"classic,consumer,share"` should be enough.

Reply:
Thanks for reviewing the PR. I have made the required changes in the latest commit